package exception;

public class ExceptionTest2 {

	public static void main(String[] args) {
		
	}

}
